apps/fluent-tester/src/E2E/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@fluentui-react-native/e2e-testing-specs",
```

Comment:
So we have a package inside a package? Feels like that might break something somewhere 😅

Reply:
Files changed 9->144 :D

Reply:
😅 I trust your judgement either way. I'm not sure what is better TBH